src/v2/view-builder/views/ov/ChallengeOktaVerifyView.js

```diff
@@ -0,0 +1,19 @@
+import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
+import AuthenticatorVerifyFooter from '../../components/AuthenticatorVerifyFooter';
+import ChallengeOktaVerifyTotpView from './ChallengeOktaVerifyTotpView';
+import ChallengeOktaVerifyFastPassView from './ChallengeOktaVerifyFastPassView';
+
+export default BaseAuthenticatorView.extend({
+  initialize () {
+    BaseAuthenticatorView.prototype.initialize.apply(this, arguments);
+
+    const currentAuthenticator = this.options?.appState?.get('currentAuthenticator');
+    const selectedChannel = currentAuthenticator?.contextualData?.selectedChannel;
+    if (selectedChannel === 'totp') {
```

Comment:
Add comment about `okta verify push` is handled by `ChallengeOktaVerifyPushView.js` because it has different form (`verify-poll`).
Intuitively I though we'd handle all channel here (totp, push and fastpass) then realize okta push gets different form.

Reply:
Yes, ultimately this will change where `totp` will be mapped to `CHALLENGE_AUTHENTICATOR` and we will need to switch between push and uv device challenge mapping to `CHALLENGE_POLL` form